Solutions/OpenCTI/Playbooks/OpenCTIPlaybooks/OpenCTI-GetIndicatorsStream/azuredeploy.json

```diff
@@ -0,0 +1,790 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Get Indicators into Sentinel from OpenCTI",
+        "description":  "This playbook fetches indicators from OpenCTI and send to Sentinel. Supported types are Domain, File, IPv4, IPv6, Account, Url.  This runs for every 10 minutes",
+        "prerequisites":  ["1. OpenCTI Custom Connector needs to be deployed prior to the deployment of this playbook under the same subscription.",
+        "2. API key. To get API Key, login into your OpenCTI instance dashboard and navigate to User profile page --> API Access.",
+        "3. OpenCTI-ImportToSentinel must be installed and keep playbook name and batchname handy"],
+        "postDeployment":  ["1. Once deployment is complete, you will need to authorize each connection.",
+        "2. Navigate to playbook --> API Connections --> Select connections one by one --> Edit API Connection --> (if required) Enter API key or credentials --> Save"],
+        "prerequisitesDeployTemplateFile":  "../../OktaCustomConnector/azuredeploy.json",
```

Comment:
Please check for okta word in all playbook. and OpenCTI-GetIndicatorsStream playboook expects Import-to-sentinel to be pre deployed other wise this playbook deployment fails , so please add that as well in prerequisite

Reply:
yeah right, currently more than 1 dependency is not supported, so I just mentioned OpenCTICustomConnector now. Other dependency, I have written in prerequisites instructions